### ISSUE TITLE
Inconsistent termination message output

### ISSUE THREAD

**Issue by mgdevstack** [kubernetes/kubernetes#70990]
**Which jobs are failing**:
https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70492/pull-kubernetes-node-e2e/110020/

**Which test(s) are failing**:
[k8s.io] Container Runtime blackbox test on terminated container should report termination message from log output if TerminationMessagePolicy FallbackToLogsOnError is set [NodeConformance]

**Since when has it been failing**:
Failing under PR #70492

**Testgrid link**:
https://testgrid.k8s.io/presubmits-kubernetes-blocking#pull-kubernetes-node-e2e&include-filter-by-regex=from%20log%20output%20if%20TerminationMessagePolicy%20FallbackToLogsOnError%20is%20set&show-stale-tests=

**Reason for failure**:
Unable to match the inconsistent termination message with an expected value.

https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70492/pull-kubernetes-node-e2e/110020/
```
/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/common/runtime.go:193
Expected
    <string>: DONE
    
to equal
    <string>: DONE
/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/common/runtime.go:161
```
https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70492/pull-kubernetes-node-e2e/110014/
```
/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/common/runtime.go:193
Expected
    <string>: DONE
to equal
    <string>: DONE
    
/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/e2e/common/runtime.go:161
```
**Anything else we need to know**:
- https://github.com/kubernetes/kubernetes/pull/70492#issuecomment-435030492

/kind failing-test

**Comment by mgdevstack** [comment]
Just for quick update.
From the latest [commit](https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70492/pull-kubernetes-node-e2e/112766/) of PR #70492, following points are observed from [testgrid report](https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/70492/pull-kubernetes-node-e2e/110020/nodelog?pod=&junit=junit_ubuntu_05.xml&wrap=on&logfiles=%2Fkubernetes-jenkins%2Fpr-logs%2Fpull%2F70492%2Fpull-kubernetes-node-e2e%2F110020%2Fartifacts%2Ftmp-node-e2e-dc52f49a-ubuntu-gke-1604-xenial-v20170420-1%2Fkubelet.log&poduid=&ns=&cID=&others=) -

**PASSING** under 
- local cluster (hack/local-up-cluster.sh)
- tmp-node-e2e-dc52f49a-coreos-beta-1883-1-0-v20180911
- tmp-node-e2e-dc52f49a-cos-stable-60-9592-84-0
- tmp-node-e2e-dc52f49a-cos-stable-63-10032-71-0

**FAILING** under
- tmp-node-e2e-dc52f49a-ubuntu-gke-1604-xenial-v20170420-1

**Comment by mgdevstack** [comment]
Seeking attention from @kubernetes/sig-node-test-failures for troubleshooting issue.

`[k8s.io] Container Runtime blackbox test when starting a container that exits should report termination message from log output if TerminationMessagePolicy FallbackToLogOnError is set [NodeConformance]` test is running successful on various platforms but not able to retrieve test's execution logs in kubelet's log.

Tried investigating issue (ref kubelet.log) from various execution platform where test is successful but couldn't find sufficient log entries.
- [k8s-testgrid.appspot.com/sig-node-kubelet](k8s-testgrid.appspot.com/sig-node-kubelet#node-kubelet-1.12&include-filter-by-regex=from%20log%20output%20if%20TerminationMessagePolicy%20FallbackToLogOnError%20is%20set)
- [k8s-testgrid.appspot.com/sig-gcp-master#gci-gke](https://k8s-testgrid.appspot.com/sig-gcp-master#gci-gke&include-filter-by-regex=from%20log%20output%20if%20TerminationMessagePolicy%20FallbackToLogOnError%20is%20set)

Seeking SIG help to troubleshoot test failure observed in PR #70492

**Comment by haiyanmeng** [comment]
@mgdevstack, there are two problems here:
1) so far the broken tests never get run because a bug in the test code (this explains why pull-kubernetes-node-e2e  never fails on our PRs), which should be fixed by #72793 . 
2) With #72793, I can reproduce your problem, and verify that the ubuntu image used in our test `ubuntu-gke-1604-xenial-v20170420-1` is broken. We will update the ubuntu image.

**Comment by mgdevstack** [comment]
Thanks @haiyanmeng for looking into this long-awaited issue.
